settings/_settings.global.scss

```diff
@@ -2,17 +2,23 @@
 //   #GLOBAL
 //   ======================================================================== */
 
-// Project-level settings and setup.
+// The global settings file contains any project-wide variables.
+
 
 // Base typographical styles and baseline grid. You need to define these values
 // in pixels: inuitcss will convert them to more appropriate units.
+
 $inuit-global-font-size:    16px !default;
 $inuit-global-line-height:  24px !default;
 
+
 // Do we need to support IE8?
+
 $inuit-oldIE: false !default;
 
+
 // Standardise some UI treatments.
+
 $inuit-global-radius:       3px !default;
```

Comment:
@csswizardry Two questions (about `$inuit-global-radius` and `$inuit-global-transition`): 

1. Are we using these anywhere in the inuit core? I see it only in button component example, but I might missed something.
2. This kinda goes into design territory (like comment says "Standardise some UI treatments"). Do we want inuit to do that? Especially when it's not used anywhere in inuit core. 

If I remember it right, this is leftover from when this file used to be `_example.settings.global.scss`. While the examples are really great in terms of educating users how to setup *their* variables, it's kinda confusing that they are part of inuit.

This is just me thinking out loud.